test/test_nn.py

```diff
@@ -1654,8 +1655,14 @@ def assign_weight():
         # This should work though
         l2.weight = Parameter(torch.randn(10, 10))
 
-    def test_clip_grad_norm(self):
-        l = nn.Linear(10, 10)
+    @parametrize_test('device_type', ('cpu', 'cuda'))
+    @parametrize_test('foreach', (False, True))
+    @parametrize_test('norm_type', (0.5, 1.5, 2, 4, 'inf'))
+    def test_clip_grad_norm(self, device_type, norm_type, foreach):
+        if device_type == 'cuda' and not TEST_CUDA:
```

Comment:
We should probably skip the CPU when TEST_CUDA is True to not replicate this test across our CPU + GPU CI configs.

Reply:
done